### ISSUE TITLE
Toggle with default value 1 show "off" in creating new

### ISSUE THREAD

**Issue by jesseint** [directus/app#2425]
Hi,

I updated my toggle default value to "1" instead of NULL.

When I created a new data, I got the page with not an active value. It shows "off" instead of "on".

After I saved, when I open it again then it will show "on".

But I guess this shouldn't be the case, if we put default 1, then creating new should show the ON toggle so it won't confuse user.

I can attach a screenshots if necessary.

Version: 8.3.0

**Comment by jesseint** [comment]
I just upgraded to 8.4.0 and this seems to already working (my default value from previous version when I put "1" now showing ON for create new entry).

However, I found another bug with this toggle in version 8.4.0.. now another one that the default value used to be 0, I wanted to change it to 1. I saved, I open again and default value is now empty. 

I did a couple of times, even log out and login in incognito but still same. Toggle couldn't save the default value even though I got a notification "Field Updated".
